apps/datahub/src/app/record/record-apis/record-apis.component.ts

```diff
@@ -75,6 +76,8 @@ export class RecordApisComponent implements OnInit {
   }
 
   openRecordApiForm(link: DatasetServiceDistribution) {
+    this.displayApiIgnForm =
```

Comment:
This could be simpliefied, as the expression already evaluates to a boolean.
```suggestion
    this.displayApiIgnForm = link.accessServiceProtocol === 'GPFDL';
```

Reply:
Yes, you're right. I made the change.